dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -895,6 +901,20 @@ def process_data_chunk(self, chunk_df, shared_data, file_row_count, sess=None, l
                 'status': 'end'
             })
 
+    @staticmethod
+    def retrieve_sub_tier_agencies(chunk_df, sess):
```

Comment:
Can this be renamed to something like `retreive_agency_codes`? Same with the resulting variable `sub_tier_agencies` -> `agency_codes`.